dhis-2/dhis-services/dhis-service-reporting/pom.xml

```diff
@@ -90,6 +90,11 @@
       <groupId>org.hisp.dhis</groupId>
       <artifactId>dhis-support-external</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.jsoup</groupId>
+        <artifactId>jsoup</artifactId>
+        <version>1.11.3</version>
+    </dependency>

```

Comment:
What are we using this library for?

Reply:
To convert interpretations/comments -now in HTML format because we use CKEditor- to plain text. We could use a regexp but seemed to more clean to use a library that used an HTML parser.

https://github.com/dhis2/dhis2-core/pull/2019/files#diff-12fe023f400f7c4d6bf945260269b5d7R293